YoCode/Program.cs

```diff
@@ -1,16 +1,13 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace YoCode
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Welcome to YoCode!");
```

Comment:
Did you mean to remove the contents of Main? I think this change should be reverted so that Main still has code in it.